### ISSUE TITLE
WebAdmin not compatible with PHP7

### ISSUE THREAD

**Issue by EPinci** [hmailserver/hmailserver#182]
Just reporting that the webadmin component does not work with PHP7 reporting that PHP5 is required.

Thank you.

**Comment by jimimaseye** [comment]
The source "required version" was manually updated by a user which then allowed full use against PHP7.0.  He then tested and ran it for a month or so and reported back no problems.  (Maybe this helps to rectify - a simple change)

```
$version = explode('.', phpversion());
   if ((int) $version[0] != 5)
   {
      echo "hMailServer WebAdmin requires PHP5.";
      die;   
   }
```
Have to be changed to

```
  $version = explode('.', phpversion());
   if ((int) $version[0] != 7)
   {
      echo "hMailServer WebAdmin requires PHP5.";
      die;   
   }
```
Full details of his change (above) and his confirmation here:  https://www.hmailserver.com/forum/viewtopic.php?p=185522#p185522

**Comment by martinknafve** [comment]
Fixed for 5.7.
